lib/tasks/event_attatchments.rake

```diff
@@ -0,0 +1,8 @@
+namespace :db do
+  desc 'Destroy event_attachments versions'
+
+  task destroy_event_attachment_versions: :environment do
+    PaperTrail::Version.where(item_type: 'EventAttachment').try(:destroy_all)
+    puts 'Version records with event_attachment have been destroyed'
+  end
+end
```

Comment:
It would make sense that rubocop catches that :)
But that's out of the scope of this PR - feel free to open a new PR to address the issue.